components/virtual-node/virtual-node-write-service-impl/src/main/kotlin/net/corda/virtualnode/write/db/impl/writer/VirtualNodeWriterProcessor.kt

```diff
@@ -232,7 +232,24 @@ internal class VirtualNodeWriterProcessor(
                 //  another database.
                 it.transaction { tx ->
                     // Retrieve virtual node info
-                    val virtualNodeInfo = virtualNodeEntityRepository.getVirtualNode(shortHashString)
+                    val virtualNodeInfo = try {
+                        virtualNodeEntityRepository.getVirtualNode(shortHashString)
+                    } catch (e: VirtualNodeNotFoundException) {
+                        logger.warn("Could no find the virtual node: $shortHashString")
+                        logger.debug { e.stackTrace.joinToString(System.lineSeparator()) }
```

Comment:
I think it would be just goo denough to do:
```suggestion
                        logger.warn("Could no find the virtual node: $shortHashString", e)
```